src/internal_events/tcp.rs

```diff
@@ -114,7 +114,7 @@ pub struct TcpEventSent {
 
 impl InternalEvent for TcpEventSent {
     fn emit_logs(&self) {
-        debug!(message = "sending event.", byte_size = %self.byte_size);
+        trace!(message = "sending event.", byte_size = %self.byte_size);
```

Comment:
All the other "sending" messages but one (https://github.com/timberio/vector/blob/e99d0c1a79ddc3be211cb66404d5d2cd2d711bd7/src/sinks/util/sink.rs#L90) use `debug!`, so this would be a convention change. I agree trace seems more reasonable, but we should plan to change all of them.

Reply:
So, should we change all "sending" `debug!` to `trace!` in one PR?

Reply:
If the single concern is this change then one PR should be fine.

Reply:
I think there were two concerns—one typo vs the debug messages that should be—and so the question was to bundle them together.

Reply:
Got it. Given these are simple, I think it's ok to bundle them.